test/e2e/compatibility/cp_compatibility_kubernetes_multizone.go

```diff
@@ -26,6 +27,22 @@ func CpCompatibilityMultizoneKubernetes() {
 	var zoneDeployOptsFuncs = KumaZoneK8sDeployOpts
 	var zoneReleaseName string
 
+	// Ensure that the upstream Kuma help repository is configured
+	// and refreshed. This is needed for helm to be able to pull the
+	// OldChart version of the Kuma helm chart.
+	BeforeSuite(func() {
+		t := NewTestingT()
+		opts := helm.Options{}
+
+		// Adding the same repo multiple times is idempotent, but causes helm to exit with an
+		// error. That's why we don't have an expectation here.
+		_, _ = helm.RunHelmCommandAndGetOutputE(t, &opts,
+			"repo", "add", "kuma", "https://kumahq.github.io/charts")
```

Comment:
Looks like adding `--force-update` might help with the error but fine either way